notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -83,7 +83,9 @@
     "import hera_pspec as hp\n",
```

Comment:
<p>I think the solution here is to basically swap Figures 6 and 7</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
6 and 7 both involve pI as written so I wasn't sure how this would solve the problem. I instead made Figure 6 use ee pol.